src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,21 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random;
+
+    public Random getRandom() {
+        return random;
+    }
+
+    public void setRandom(Random random) {
+        this.random = random;
+    }
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        random = new Random();
```

Comment:
Assign a value to random immediately or in the constructor

Reply:
Not fixed.
Initialize either through the constructor or in a variable directly:
private Random random = new Random();

Reply:
fix this